website/source/docs/post-processors/exoscale-import.html.md

```diff
@@ -0,0 +1,89 @@
+---
+description: |
+    The Packer Exoscale Import post-processor takes an image artifact
+    from various builders and imports it to Exoscale.
+layout: docs
+page_title: 'Exoscale Import - Post-Processors'
+sidebar_current: 'docs-post-processors-exoscale-import'
+---
+
+# Exoscale Import Post-Processor
+
+Type: `exoscale-import`
+
+The Packer Exoscale Import post-processor takes an image artifact from
+various builders and imports it to Exoscale.
```

Comment:
Which specific artifact types can be imported? It'll be important for users to know what artifacts are allowed.  

You may want to add a test for this into the code itself. Example: 

https://github.com/hashicorp/packer/blob/master/post-processor/docker-import/post-processor.go#L48-L56

Reply:
Thanks, fixed.